### PR TITLE
Add more POIs in Canada

### PR DESCRIPTION
**Pull request by andrewpmk**
Added POIs:
- Access Storage
- Active Green + Ross
- Ali Baba's
- Becker's
- Big Bear/Big Bee
- Bikini Village
- BioPed
- Blinds to Go
- Bouclair Home
- Bourbon St. Grill
- BrokerLink
- Burrito Boyz
- Maison Birks
- Mister Transmission
- Mobile Klinik
- Moxie's
- Quickie
- Sirens
- South Street Burger
- Stinson
- Sun Life Financial
- The Works
- Trade Secrets
Updated POIs:
- 9Round
- Allstate
- Alterna Savings
- Boathouse
- Nails For You

### CODE REVIEW

File: data/brands/shop/window_blind.json [hunk 1]
```diff
@@ -4,6 +4,17 @@
     "exclude": {"generic": ["^window blind$"]}
   },
   "items": [
+    {
+      "displayName": "Blinds to Go",
+      "id": "blindstogo-a6607a",
+      "locationSet": {"include": ["ca"]},
```
Comment (AntiCompositeNumber) [inline data/brands/shop/window_blind.json:10]:
```suggestion
      "locationSet": {"include": ["ca-on.geojson", "us-ct.geojson", "us-de.geojson", "us-ma.geojson", "us-md.geojson", "us-nj.geojson", "us-ny.geojson", "us-pa.geojson", "us-va.geojson"]},
```
per https://www.blindstogo.com/en/stores

Comment (AntiCompositeNumber) [inline data/brands/shop/window_blind.json:10]: Please also add an entry for Quebec per https://www.blindstogo.com/fr/magasins

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]
In the future, smaller, more focused pull requests are easier to review.

File: data/brands/shop/interior_decoration.json [hunk 2]
```diff
@@ -17,6 +17,17 @@
...
+      "displayName": "Bouclair Home",
+      "id": "bouclairhome-aa1ae6",
+      "locationSet": {"include": ["ca"]},
+      "tags": {
+        "brand": "Bouclair",
+        "brand:wikidata": "Q20638837",
+        "name": "Bouclair Home",
+        "shop": "interior_decoration"
+      }
+    },
```
[the remark this answers is not in the document]
Comment (Cj-Malone) [reply data/brands/shop/interior_decoration.json:30]: In theory `name` should be the [facia](https://www.google.com/maps/place/Bouclair+Home/@43.841389,-79.4211637,3a,75y,350.68h,93.34t/data=!3m7!1e1!3m5!1sTsTvVsXs7ve9J7Ix9ZEj2Q!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DTsTvVsXs7ve9J7Ix9ZEj2Q%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D341.89697%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192!4m10!1m2!2m1!1sBouclair+Home!3m6!1s0x882b2b836f0272ff:0x1bbb23ec256ca6e4!8m2!3d43.8415112!4d-79.4212305!15sCg1Cb3VjbGFpciBIb21lIgOIAQFaDyINYm91Y2xhaXIgaG9tZZIBD2Z1cm5pdHVyZV9zdG9yZeABAA!16s%2Fg%2F1tjl6jt0?entry=ttu) and `brand` the ["brand"](https://www.bouclair.com/en/).

However OSM and NSI aren't prefect in this.

File: data/brands/shop/convenience.json [hunk 3]
```diff
@@ -617,6 +628,29 @@
...
     },
+    {
+      "displayName": "Big Bear",
+      "id": "bigbear-e144dd",
+      "locationSet": {
+        "include": ["ca-on.geojson"]
+      },
+      "tags": {
+        "brand": "Big Bear",
+        "brand:wikidata": "Q123409887",
```
[the remark this answers is not in the document]
Comment (andrewpmk) [reply data/brands/shop/convenience.json:639]: Looks like they are separate chains https://www.reddit.com/r/Hamilton/comments/4bw8rt/big_bear_vs_big_bee_stores/
They're chains of convenience stores near Hamilton Ontario (also Niagara Falls, St Catharines, Kitchener etc.)

File: data/brands/shop/interior_decoration.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrewpmk) [reply data/brands/shop/interior_decoration.json:30]: They're different brands of store operated by the same company. NSI keeps asking me to change "Bouclair Home" stores to "Bouclair" which is incorrect.
We have the same issue with "The Brick" vs "The Brick Mattress Store".

File: data/brands/leisure/fitness_centre.json [hunk 4]
```diff
@@ -61,8 +61,39 @@
     },
     {
       "displayName": "9Round",
-      "id": "9round-809afb",
-      "locationSet": {"include": ["us"]},
+      "id": "9round-23f8d6",
+      "locationSet": {
+        "include": [
```
[the remark this answers is not in the document]
Comment (andrewpmk) [reply data/brands/leisure/fitness_centre.json]: Changed to 001

File: data/brands/shop/convenience.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (andrewpmk) [reply data/brands/shop/convenience.json]: Removed Big Bear. Looks like it is not the same chain https://www.reddit.com/r/Hamilton/comments/4bw8rt/big_bear_vs_big_bee_stores/

File: data/brands/shop/interior_decoration.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrewpmk) [reply data/brands/shop/interior_decoration.json]: Added French entry

**Comment by andrewpmk** [comment]
Updated pull request with suggested changes.
Removed Big Bear because it appears to be different from Big Bee.
Added French entries for Bouclair Home and Blinds to Go.

File: data/brands/shop/window_blind.json [hunk 5]
```diff
@@ -52,6 +64,20 @@
...
+    {
+      "displayName": "Le Marché du Store",
+      "id": "lemarchedustore-4eac4b",
+      "locationSet": {
+        "include": ["ca-qc.geojson"]
+      },
+      "matchNames": ["blinds to go"],
+      "tags": {
+        "brand": "Le Marché du Store",
+        "brand:wikidata": "Q113686682",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/window_blind.json:76]:
```suggestion
        "brand:wikidata": "Q123409913",
```

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]

**Review by AntiCompositeNumber — APPROVED** [review]

**Comment by andrewpmk** [comment]
Updated brand:wikidata of Blinds to Go to Q123409913

**Pull request merged**